code/modules/events/immovable_rod.dm

```diff
@@ -137,31 +137,37 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 		qdel(src)
 		qdel(other)
 
-/obj/effect/immovablerod/proc/penetrate(mob/living/L)
-	L.visible_message("<span class='danger'>[L] is penetrated by an immovable rod!</span>" , "<span class='userdanger'>The rod penetrates you!</span>" , "<span class='danger'>You hear a CLANG!</span>")
-	if(ishuman(L))
-		var/mob/living/carbon/human/H = L
-		H.adjustBruteLoss(160)
-	if(L && (L.density || prob(10)))
-		L.ex_act(EXPLODE_HEAVY)
-
-/obj/effect/immovablerod/attack_hand(mob/living/user)
+/obj/effect/immovablerod/proc/penetrate(mob/living/user)
```

Comment:
You do know that wizard rods override the normal penetrate() proc for immovable rods, right? So this PR actually makes wizard rods un-suplex-able.

Reply:
hopefully its fixed now